backend/tests/v5/unit/processors/users.test.js

```diff
@@ -60,10 +65,15 @@ const getUserByUsernameMock = UsersModel.getUserByUsername.mockImplementation((u
 const updateUserByUsernameMock = UsersModel.updateProfile.mockImplementation(() => {});
 const updatePasswordMock = UsersModel.updatePassword.mockImplementation(() => {});
 const updateResetPasswordTokenMock = UsersModel.updateResetPasswordToken.mockImplementation(() => {});
+const addUserMock = UsersModel.addUser.mockImplementation(() => {});
```

Comment:
I don't think we need any of these mocks if you're just mocking it to do nothing (try the test and remove all of this declarations and just use `UsersModel.<func>` directly in the code.